lib/mongoid/document.rb

```diff
@@ -98,9 +98,11 @@ def identity
     #   Person.new(:title => "Sir")
     #
     # @param [ Hash ] attrs The attributes to set up the document with.
+    # @param [ true | false ] defer_callbacks Flag specifies whether callbacks
+    #   should be run.
     #
     # @return [ Document ] A new document.
-    def initialize(attrs = nil)
+    def initialize(attrs = nil, defer_callbacks: false)
```

Comment:
Is this okay? This is a public function so I'm not sure if I could add to the method signature.